bad_packages.md

```diff
@@ -0,0 +1,43 @@
+A few errors found by randomly installing some lwt packages.
+
+All seem to point to libraries using lwt.unix but only linking to lwt.
```

Comment:
To support camlp4 properly (for the time being) we will need to hand maintain a META file anyway.

Personally, I would just break those packages as they are strictly wrong, and can cause upstream problems for their users anyway.  But maybe that's just not playing very nicely...